marbles/core/tests/test_marbles.py

```diff
@@ -586,6 +604,12 @@ def test_note_wrapping(self):
         lines = e.note.split('\n')
         self.assertTrue(any(len(line) > 75 for line in lines))
 
+        with self.assertRaises(ContextualAssertionError) as ar:
+            self.case.test_indented_note()
+        e = ar.exception
+        lines = e.note.split('\n')
+        self.assertTrue(all(line.lstrip() == line[1:] for line in lines))
```

Comment:
This test means that every line starts with exactly one tab (and no more). Should I add a comment explaining?

Reply:
What if, instead of adding a new test method with an indented-in-source note and a separate assertion, we indented all of the notes in the example test case test methods and ensured that the assertions we already have still pass? I say this because there are a couple of kinds of notes that aren't covered by this test method and assertion (e.g., notes with indented lists, notes with paragraphs). I guess in a perfect world we would have all notes in indented and not indented form, but that seems like overkill for our usage of `textwrap`. I'm proposing we indent all test notes because more people will want to indent notes in source, but I have little data to back that up.

What do you think?

Reply:
If we want to keep this assertion, we could make it more explicit with:
```
self.assertTrue(all(line.lstrip('\t') == line[1:] for line in lines))
```

This would work for the test note you added, but should fail for a note with, e.g., an indented list I think.

Reply:
I am not completely comfortable with making a test check more than one feature, but for what you are saying it makes sense. Yes, ideally we would have double of all the tests but it is overkill. I'll just change the other tests.